lzy/lzy-service/src/main/java/ai/lzy/service/graph/GraphBuilder.java

```diff
@@ -293,6 +293,20 @@ private TaskDesc buildTaskWithZone(String taskId, LWF.Operation operation,
 
         env.setDockerImage(operation.getDockerImage());
 
+        if (operation.hasDockerCredentials()) {
+            env.setDockerCredentials(LME.DockerCredentials.newBuilder()
+                .setUsername(operation.getDockerCredentials().getUsername())
+                .setPassword(operation.getDockerCredentials().getPassword())
+                .setRegistryName(operation.getDockerCredentials().getRegistryName())
+                .build());
+        }
+
+        if (operation.getDockerPullPolicy() == LWF.Operation.DockerPullPolicy.ALWAYS) {
+            env.setDockerPullPolicy(LME.DockerPullPolicy.ALWAYS);
+        } else {
+            env.setDockerPullPolicy(LME.DockerPullPolicy.IF_NOT_EXISTS);
```

Comment:
Don't do that, plz :)

we should process the input data "honestly" and issue an error in the case of an invalid config